ONTOLOGY_GUIDE.md

```diff
@@ -273,137 +273,75 @@ Because we believe that it is important to ensure at least some minimal level of
 1. **Crossing domains / ranges**  - If one property is a sub of another, then the domains (respectively ranges) should not be subClasses in the opposite direction. See [testHygiene0002.sparql](https://github.com/edmcouncil/fibo/blob/master/etc/testing/hygiene/testHygiene0002.sparql), [testHygiene0003.sparql](https://github.com/edmcouncil/fibo/blob/master/etc/testing/hygiene/testHygiene0003.sparql)
 1. **Labels and Definitions** - Every Class and Property defined in FIBO must have an rdfs:label and a skos:definition. See [testHygiene0004.sparql](https://github.com/edmcouncil/fibo/blob/master/etc/testing/hygiene/testHygiene0004.sparql)
 1. **Ontology Metadata** - Every Ontology defined in FIBO must have a rdfs:label, sm:copyright, dct:license, dct:abstract. See [testHygiene0005.sparql](https://github.com/edmcouncil/fibo/blob/master/etc/testing/hygiene/testHygiene0005.sparql)
+1. **Special Characters** - The set of characters allowed in literals is limited to the alphanumeric characters, punctuation marks, and diacritic characters used in the languages FIBO uses. See [testHygiene0114.sparql](https://github.com/edmcouncil/fibo/blob/master/etc/testing/hygiene/testHygiene0114.sparql)
+1. **Polysemous elements** - We should avoid using polysemous names, in particular no label can name more than one owl:Class or rdf:Property. See [testHygiene1067.sparql](https://github.com/edmcouncil/fibo/blob/master/etc/testing/hygiene/testHygiene1067.sparql)
+1. **Circular Definitions** - No Fibo definition should be circular, i.e., its definiens cannot contain any resource defined by means of its definiendum. See [testHygiene1068.sparql](https://github.com/edmcouncil/fibo/blob/master/etc/testing/hygiene/testHygiene1068.sparql)
```

Comment:
I think use of "any resource defined by" is not really correct - we're just saying that a definition cannot contain a label (including synonyms) of the resource being defined.
Again, the use of Latin is unlikely to be understood.
BTW FIBO should be in caps